source/MRMesh/MRMeshTrimWithPlane.h

```diff
@@ -5,6 +5,16 @@
 #include <functional>
 namespace MR
 {
+/// \return All vertices on the positive side of the plane
+/// \param pc Input point cloud that will be cut by the plane
+/// \param plane Input plane to cut mesh with
+MRMESH_API VertBitSet subdivideWithPlane( const PointCloud& pc, const Plane3f& plane );
```

Comment:
does not really subdivide mb better name it `findHalfSpacePoints` or something like this